flink-core/src/test/java/org/apache/flink/api/common/typeutils/TypeSerializerUpgradeTestBase.java

```diff
@@ -263,14 +268,17 @@ void generateTestSetupFiles(
             DataOutputSerializer testDataOut = new DataOutputSerializer(INITIAL_OUTPUT_BUFFER_SIZE);
             priorSerializer.serialize(testSpecification.setup.createTestData(), testDataOut);
             writeContentsTo(
-                    getGenerateDataFilePath(testSpecification), testDataOut.getCopyOfBuffer());
+                    getGenerateDataFilePath(testSpecification, testSpecification.flinkVersion),
```

Comment:
```suggestion
                    getGenerateDataFilePath(testSpecification),
```
The second parameter is obsolete.